### ISSUE TITLE
testregistry incomplete/inconsistent data

### ISSUE THREAD

**Issue by MarcCharlebois** [openconfig/featureprofiles#3439]
**Describe the bug**
testregistry incomplete and inconsistent data

**To Reproduce**
N/A

**Expected behavior**
N/A

**Logs**
N/A

**Network device platform (please complete the following information):**
N/A

**Additional context**
Looking at the proto for testregistry - there seem to be some inconsistencies with the current values populated and the proto. Please see below examples.

- Authz-1 Doesn't follow the the regex listed (not sure if the regex should be adapted to fit the tests more loosely, or if the test IDs should be restricted to the regex).
https://github.com/openconfig/featureprofiles/blob/f11401cb96f286fe7ec76ea68ac590f58242aaba/proto/testregistry.proto#L32-L35
https://github.com/openconfig/featureprofiles/blob/f11401cb96f286fe7ec76ea68ac590f58242aaba/testregistry.textproto#L76
- Certain test IDs are repeated in the description (shouldn't be necessary if the we already have the ID) https://github.com/openconfig/featureprofiles/blob/f11401cb96f286fe7ec76ea68ac590f58242aaba/testregistry.textproto#L834-L835
- Do we expect each test to be fully populated with `Description`, `ID`, `Readme`, `Exec`?

**Comment by dplore** [comment]
Yes, we want testregistry to provide an id, description and link to the relevant README.

I've created an internal tracking issue to add a CI check (and a generator for testregistry) to resolve these issues.
